### PR TITLE
MM-42877: Fix TestGetFileHeaders

### PR DESCRIPTION
**Pull request by agnivade**
The issue seems to be that different distros
have different set of included mime types
built-in. Go looks into those directories
and it looks like our CI systems don't have
those files.

https://mattermost.atlassian.net/browse/MM-42877

```release-note
NONE
```

### CODE REVIEW

**Comment by agnivade** [comment]
I expect this to fail .. let's see.

**Review by isacikgoz — APPROVED** [review]
Nice!

File: api4/file_test.go [hunk 1]
```diff
@@ -817,7 +818,11 @@ func TestGetFileHeaders(t *testing.T) {
 	t.Run("txt", testHeaders(data, "test.txt", "text/plain", false, false))
 	t.Run("html", testHeaders(data, "test.html", "text/plain", false, false))
 	t.Run("js", testHeaders(data, "test.js", "text/plain", false, false))
-	t.Run("go", testHeaders(data, "test.go", "application/octet-stream", false, false))
+	if runtime.GOOS == "linux" || runtime.GOOS == "darwin" {
```
Comment (isacikgoz) [inline api4/file_test.go:821]: Maybe we can do this just for macos.
```suggestion
	if runtime.GOOS == "darwin" {
```

Comment (agnivade) [reply api4/file_test.go:821]: It fails on my machine as well. The proper fix is to set those files in the docker container by upgrading the base dependency. Might be tricky.

**Pull request merged**